ObjCVoiceCallKitQuickstart/ViewController.m

```diff
@@ -602,6 +609,7 @@ - (void)performAnswerVoiceCallWithUUID:(NSUUID *)uuid
     }
     
     self.callInvite = nil;
+    [self incomingPushHandled];
```

Comment:
I like this change. But I wonder will it move the app into background suspended if connect takes longer? One way to confirm is switch on the "very bad" network on LTE, and answer call in background and see what happens.

Reply:
If the allowed wake time is that short before the app is put to bg-suspended, then this could've happened without the extreme network setup. The only way to resolve the concern is to fulfill the completion when the call is connected or failed to connect.

Reply:
In fact I tried both putting the app to the background or even locked the phone - the heart-beat debugging log resuming printing upon receiving the incoming push and did not stop whether the completion is fulfilled or not, until the call is ended and reported to CallKit.

Reply:
Based on the discussion with @piyushtank, tried the above scenarios but this time with the app terminated - got the same result showing that calling the incoming-push completion handler won't put the app to the background-suspended state. App will only be suspended when the call is ended.